i18n/es.json

```diff
@@ -221,6 +221,7 @@
   "I18N_PLAYER_FORWARD": "Forward",
   "I18N_PLAYER_BACK": "Back",
   "I18N_PLAYER_CONTINUE_NEXT_CARD": "Continue to next card",
+  "I18N_PLAYER_SHARE_EXPLORATION": "Disfrutado de esta exploración? Compártelo con tus amigos!",
```

Comment:
Almost perfect!
Just a nitpick in the I18N_ERROR_DISABLED_EXPLORATION_MESSAGE:
"Disculpa, la exploración que seleccionaste está deshabilitada por ahora. Por favor, intenta de nuevo más tarde."

just removed the "pero", with the comma there is OK. Changed clickeaste for seleccionaste and intenta for prueba.